content/en/posts/inner-misalignment.md

```diff
@@ -0,0 +1,11 @@
+Intermediate
+
+Inner misalignment arises when there's a divergence between the objective we give an AI system and how the system internally learns to achieve that objective. In essence, the AI might develop strategies or behaviors during its training that are not explicitly intended by its creators.
+Think of it like teaching a child to get good grades in school. Your intention is for the child to study hard, understand the material, and perform well on tests. However, the child might realize that copying from a classmate or using hidden notes could also lead to good grades. The child's behavior (cheating) is misaligned with your intended method of achieving the objective (genuine study).
+In the AI domain, machine learning models can sometimes find shortcuts or exploit loopholes in the data or the specified objective to achieve desired results, even if those shortcuts contradict our intentions.
+For instance, if you train a machine learning model to identify zebras in images, the model might not actually learn to recognize the shape and features of zebras. Instead, if most training images of zebras were taken in the African savannah during daytime, the model might just learn to identify the savannah's grassy background or the specific lighting of those photos as indicators of a zebra's presence. Thus, when presented with a nighttime picture of a zebra or one in a different setting, the model might fail to recognize it.
+
+Relation to Other Terms
+
+While inner misalignment concerns an AI's learned behavior diverging from its specified objective, outer misalignment is about the mismatch between the human's true intentions and the objective given to the AI.
```

Comment:
Just to add, here I agree with the definition as it meets my understanding of inner and outer misalignment. I just think that the examples before don't quite match this distinction. :)

Reply:
General additional note: I added this post with several adjustments to the outer misalignment PR, so I can properly crosslink between the two posts. So if that one gets through, we can close this PR as a ~duplicate.